pkg/indicator/v2/momentum/rsi.go

```diff
@@ -28,7 +29,7 @@ func RSI2(source types.Float64Source, window int) *RSIStream {
 func (s *RSIStream) Calculate(_ float64) float64 {
 	var gainSum, lossSum float64
 	var sourceLen = s.source.Length()
-	var limit = min(s.window, sourceLen)
+	var limit = indicator.Min(s.window, sourceLen)
```

Comment:
try not to use functions from the indicator v1 package, it can cause cyclic import.

you can simply copy the function code to v2, it's fine~

Reply:
good call